generator/CPP11/include_v2.0/msgmap.hpp

```diff
@@ -66,70 +63,47 @@ class MsgMap {
 
 // implementation
 
-template<typename _T, typename _Tin>
-void mavlink::MsgMap::msg_swap_memcpy(_T &buf, _Tin data)
-{
-	if (std::is_floating_point<_Tin>::value) {
-		buf = *static_cast<const _T *>(static_cast<const void *>(&data));
-	} else {
-		buf = data;
-	}
```

Comment:
Actually you can just put `memcpy()` instead of that deref.

Reply:
Indeed, that's true. I changed the design, because previously both `MsgMap::msg_swap_memcpy` and `MsgMap::operator<<` modified `_MAV_PAYLOAD_NON_CONST` (see [here](https://github.com/ArduPilot/pymavlink/blob/master/generator/CPP11/include_v2.0/msgmap.hpp#L97) and [here](https://github.com/ArduPilot/pymavlink/blob/master/generator/CPP11/include_v2.0/msgmap.hpp#L108)).

Reply:
`msg_swap_memcpy()` is just a way to avoid copy-paste in `<<`.